test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateTFMs.cs

```diff
@@ -16,7 +16,47 @@ namespace Microsoft.DotNet.ProjectJsonMigration.Tests
     public class GivenThatIWantToMigrateProjectFramework : TestBase
     {
         [Fact]
-        public void Migrating_netcoreapp_project_Populates_TargetFrameworkIdentifier_and_TargetFrameworkVersion()
+        public void Migrating_netcoreapp_project_Does_not_populate_TargetFrameworkIdentifier_and_TargetFrameworkVersion()
+        {
+            var testDirectory = Temp.CreateDirectory().Path;
+            var testPJ = new ProjectJsonBuilder(TestAssetsManager)
+                .FromTestAssetBase("TestAppWithRuntimeOptions")
+                .WithCustomProperty("buildOptions", new Dictionary<string, string>
+                {
+                    { "emitEntryPoint", "false" }
+                })
+                .SaveToDisk(testDirectory);
+
+            var projectContext = ProjectContext.Create(testDirectory, FrameworkConstants.CommonFrameworks.NetCoreApp10);
+            var mockProj = ProjectRootElement.Create();
+
+            var testSettings = new MigrationSettings(testDirectory, testDirectory, "1.0.0", mockProj);
```

Comment:
I think calling these migrationSettings and migrationInputs would be more meaningful.